policies/compound_library.rego

```diff
@@ -0,0 +1,23 @@
+package xchemlab.compound_library
+
+import data.xchemlab
+import future.keywords.if
```

Comment:
Importing `rego.v1` is preferred to `futures.x` as it ops into all of the breaking changes that will come with OPA v1

```suggestion
import rego.v1
```

Reply:
I guess future.keywords.if is used for other services as well. But I've changed for this service

Reply:
Ah, probably want to change it for all of them, but I can do that at a later date